### ISSUE TITLE
Crashed: Cannot invoke "com.patrigan.faction_craft.capabilities.factionentity.FactionEntity.getFaction()" because "sourceCap" is null

### ISSUE THREAD

**Issue by LegendaryTot** [Patrigan/FactionCraft#3]
Game crashed with error:
```
java.lang.NullPointerException: Cannot invoke "com.patrigan.faction_craft.capabilities.factionentity.FactionEntity.getFaction()" because "sourceCap" is null
```

[latest.log](https://pastebin.com/WiEwzaLR)

Versions:

- Minecraft: 1.18.1
- Forge: 39.0.57
- FactionCraft: 1.0.1

**Comment by Patrigan** [comment]
Difficult one. Lots of info to sift through. I could quickly implement a workaround, but it feels like the incorrect final solution.

**Comment by Patrigan** [comment]
Should be resolved with a work around in 1.0.2. Released now on CF!
